cdap-proto/src/main/java/io/cdap/cdap/proto/NamespaceMeta.java

```diff
@@ -188,6 +214,52 @@ public Builder setConfig(Map<String, String> configMap) {
       return this;
     }
 
+    public Builder setRepoProvider(String provider) {
+      this.repoProvider = provider;
+      return this;
+    }
+
+    public Builder setRepoLink(String repoLink) {
```

Comment:
some of your variable names have 'repo' at front, some do not. Can you make them consistent?